packages/dataclass-editor/src/components/dataclass/data/dataclass-utils.ts

```diff
@@ -3,3 +3,15 @@ import type { DataClass } from './dataclass';
 export const isEntityClass = (dataClass: DataClass) => {
   return !!dataClass.entity;
 };
+
+export const newFieldName = (dataClass: DataClass) => {
+  const takenNames = dataClass.fields.map(field => field.name);
+  const baseName = 'newAttribute';
+  let newName = baseName;
+  let index = 2;
+  while (takenNames.includes(newName)) {
+    newName = `${baseName}${index}`;
+    index++;
+  }
+  return newName;
+};
```

Comment:
I'm not sure if this method makes much sense....
I mean is there a usecase where I want to create many new attributes which are called "newAttribute"?
I would keep it simple and simply keep "newAttribute" as init value, but if someone created this attribute then the dialog will show a warning that this field already exist.